docs/ECLLanguageReference/ECLR_mods/Recrd-ScopeNLogicalFilenames.xml

```diff
@@ -76,6 +76,18 @@ FileY := DATASET('~Training::SomeDir::SomeFileOut2',RecStruct,THOR);</programlis
     read-only reference to remote files (both logical files and
     superfiles).</para>
 
+    <para><variablelist>
+        <varlistentry>
+          <term>NOTE:</term>
+
+          <listitem>
+            <para>If LDAP authentication is enabled on the foreign Dali, the
+            user's credentials and permissions are verified before processing
+            the file access request.</para>
```

Comment:
Might want to say "may be verified".  If LDAP is enabled, un/pw is verified.  But you also have to take the additional step to set enableScopeScans=true to enforce permissions